flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinator.java

```diff
@@ -91,9 +92,6 @@
 
 	// ------------------------------------------------------------------------
 
-	/** Coordinator-wide lock to safeguard the checkpoint updates. */
-	private final Object lock = new Object();
```

Comment:
nit: some comments like L173 and L176 I think need to be updated, as there is no "synchronized" scope anymore but just single threaded?

Reply:
Sure, nice point!